README.md

```diff
@@ -250,12 +255,80 @@ rsconnect write-manifest notebook my-notebook.ipynb
 
 > **Note:** manifests for static (pre-rendered) notebooks cannot be created.
 
+### API/Application Deployment Options
+
+There are a variety of options available to you when deploying a Python WSGi-style
+API or a Dash-style application.  All options below apply equally to `api` and `dash`
```

Comment:
I'm not sure I know what "Dash-style" means. flask-style or flask based make sense though.

Reply:
Dropped "-style"